Server/Day1_Basic_Rest_API/README.md

```diff
@@ -0,0 +1,19 @@
+# Basic_Rest_API
+>  RESTful API is a method of allowing communication between a web-based client and server 
```

Comment:
ไปเพิ่ม . มา

Reply:
RESTful API is a method of allowing communication between a web-based client and server.